### DISCUSSION TITLE
appearance of time

### DISCUSSION THREAD

**Discussion by saba1996** [YetiForceCompany/YetiForceCRM#16878] [General] [answered]
Can I display the creation date and the date of each change instead of the number of days?

![image](https://user-images.githubusercontent.com/56167014/218084516-b9e8a3d0-77a8-41cc-b8ec-58aaf747cb97.png)

Version 6.1.0

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10198654/218127305-b172b0d3-93b4-425c-87af-653d072ab8b4.png)
